src/main/java/core/basesyntax/controllers/ConsoleHandler.java

```diff
@@ -0,0 +1,77 @@
+package core.basesyntax.controllers;
+
+import core.basesyntax.dao.BetDao;
+import core.basesyntax.dao.LotteryDao;
+import core.basesyntax.lib.Inject;
+import core.basesyntax.model.Bet;
+import core.basesyntax.model.Lottery;
+
+import java.util.Scanner;
+
+public class ConsoleHandler {
+
+    private static Scanner scanner = new Scanner(System.in);
+    private static String command;
```

Comment:
let's move the variables `scanner` and `command` in the methods

Reply:
done